azure-pipelines-docs-integration.yml

```diff
@@ -88,11 +89,15 @@ stages:
               secureFile: 'superconductive-service-acct.json'
               retryCount: '2'
 
+        - script: |
+            sqlcmd -U sa -P "ReallyStrongPwd1234%^&*" -Q "CREATE DATABASE test_ci;" -o create_db_output.txt
+          displayName: 'Create MSSQL database test_ci'
+
         - script: |
             pip install pytest pytest-azurepipelines
             # TODO enable spark tests
             # TODO enable sqlalchemy once cloud resources are working again
-            pytest -v --docs-tests -m docs --no-spark --mysql tests/integration/test_script_runner.py
+            pytest -v --docs-tests -m docs --no-spark --mysql --mssql tests/integration/test_script_runner.py
```

Comment:
@Shinnnyshinshin I'm wondering if the Azure authentication errors I'm getting on integration tests have something to do with this comment: `# TODO enable sqlalchemy once cloud resources are working again`? Are we not testing Postgres? Isn't MySQL using SQL Alchemy?

`azure.core.exceptions.ClientAuthenticationError: Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.`

Reply:
So sorry this comment should be removed.             

`# TODO enable sqlalchemy once cloud resources are working again`. 

It was part of an earlier set of errors where we were having Snowflake and BigQuery tests fail. 😅